### DISCUSSION TITLE
Google Analytics

### DISCUSSION THREAD

**Discussion by armanaxf** [CaiJimmy/hugo-theme-stack#791] [Q&A] [answered]
How can I add Google Analytics to my site using Stack? Looks to be supported by both Hugo and this theme but can't figure out what to add and where?

Thanks,

**Comment by rmdhnreza** [comment, marked as answer]
You can add your Google Analytics code to this line https://github.com/CaiJimmy/hugo-theme-stack/blob/aae9cf67db8661e86996f8808baf4f71c472d911/exampleSite/config.yaml#L27-L30

or you can copy a script from Google Analytics & paste that script to `/layouts/partials/head/custom.html` or `/layouts/partials/footer/custom.html`
